packages/xchain-thorchain/__tests__/txs.ts copy.bu

```diff
@@ -0,0 +1,30 @@
+import { TxsPage } from '@xchainjs/xchain-client'
+import { Client } from '../src/client'
+
+const phrase = 'rural bright ball negative already grass good grant nation screen model pizza'
+
+const client = new Client({ phrase, network: 'mainnet' })
+
+const _example = async () => {
+  // const client = new Client({ phrase, network: 'mainnet' })
```

Comment:
delete this line.
also this file name eds with 'copy.bu' probably an error?

Reply:
Ah, this file has been committed by an accident :sweat_smile: